src/process.c

```diff
@@ -23,6 +24,12 @@ struct list ready_list = {0,0};
 struct list grave_list = {0,0};
 struct process *processes[MAX_PID] = {0};
 
+struct mount {
```

Comment:
I'm not yet totally sure that this is our final model of namespace management, but it will do for now.
Is the namespace inherited by child processes?